### PR TITLE
Main page prototype

### PR DESCRIPTION
**Pull request by katiehsieh**
Includes:
- Starter project with example Flask server and React app integration
- Home page
- Header component
- WelcomeMessage component
- TrendingTopics component
- MessageBox component
- ChatMessage component

### CODE REVIEW

**Review by zimo27 — APPROVED** [review]
Overall looks nice, good job Katie. 
- Some css files under styles still have `.module.` somehow. 
- I committed an update mistakenly to your branch, apologies

**Comment by champion-chenpeng** [comment]
looks good to me. But I think it is better to combine current two chat messages in the welcome page into a component called "instructions".
image

File: frontend/src/pages/Home.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+          <ChatMessage
+            role="bot"
+            text="To schedule your first appointment with CAPS, you can call CAPS at
+            (858) 534-3755, schedule an appointment through MyStudentChart,
+            or visit the Central Office at Galbraith Hall 190."
+          />
+        </div>
+      )}
+    
+      <div>
```
Comment (champion-chenpeng) [inline frontend/src/pages/Home.js:46]: I also recommend consolidating all elements pertaining to user prompts into a single component, thus maintaining Home.js as a pure page devoid of any data processing.

File: frontend/src/pages/Home.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    console.log(newPrompt);
+    setUserPrompts([...userPrompts, newPrompt]); // Add the new message to the state
+  };
+
+  return (
+    <div className={styles.container}>
+      <Header />
+      <WelcomeMessage />
+      {!isTyping && (
+        <div>
```
Comment (champion-chenpeng) [inline frontend/src/pages/Home.js:32]: I think it is better to separate this part into InstructionMessage component

File: frontend/src/pages/Home.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+            text="To schedule your first appointment with CAPS, you can call CAPS at
+            (858) 534-3755, schedule an appointment through MyStudentChart,
+            or visit the Central Office at Galbraith Hall 190."
+          />
+        </div>
+      )}
+    
+      <div>
+        {userPrompts.map(prompt => (
+          <ChatMessage role={prompt.role} text={prompt.text} />
```
Comment (champion-chenpeng) [inline frontend/src/pages/Home.js:48]: The XSS attack may happen in passing prompt.text to ChatMessage

**Pull request merged**